chianft/util/mint.py

```diff
@@ -114,10 +114,13 @@ async def create_spend_bundles(
         spend_bundles = []
         if mint_from_did:
             did = await self.wallet_client.get_did_id(wallet_id=self.did_wallet_id)
+            did_cr = await self.wallet_client.get_did_info(
+                coin_id=did["coin_id"], latest=True
+            )
             did_coin_record: Optional[
                 CoinRecord
             ] = await self.node_client.get_coin_record_by_name(
-                bytes32.from_hexstr(did["coin_id"])
+                bytes32.from_hexstr(did_cr["latest_coin"])
             )
             assert isinstance(did_coin_record, CoinRecord)
```

Comment:
same here `is not None`? seems more corret?

Reply:
have changed this everywhere now